base/standard/src/main/java/org/openscience/cdk/tools/manipulator/ReactionManipulator.java

```diff
@@ -539,4 +529,64 @@ public static Set<IBond> findMappedBonds(IReaction reaction) {
         }
         return mapped;
     }
+
+    /**
+     * Convenience method to perceive atom types for all {@link IAtom IAtoms} of all components of the provided {@link IReaction}.
```

Comment:
Is `{@link IAtom IAtoms}` what you meant to write?

Reply:
Yes, it is.
This links to IAtom but uses the text IAtoms for the link.